ui/app/styles/components/global-flash.scss

```diff
@@ -0,0 +1,76 @@
+.global-flash {
```

Comment:
There are [existing notification styles](https://github.com/hashicorp/nomad/blob/master/ui/app/styles/core/notification.scss) that at the very least the flash notifications should look like. Ideally they would use the same styles via some sass pattern or composition.